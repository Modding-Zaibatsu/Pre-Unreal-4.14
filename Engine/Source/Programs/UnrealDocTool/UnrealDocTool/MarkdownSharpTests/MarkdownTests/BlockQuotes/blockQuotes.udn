> Block quote first.
> 1. List in quote.
> 1. Second list item.
> 1. Third list item.
>
> > Nested quote
> > 
> > Nested quote second paragraph.
> > 
> > * Nested quote list.
> > * Second bullet.
> 
> Ending paragraph.