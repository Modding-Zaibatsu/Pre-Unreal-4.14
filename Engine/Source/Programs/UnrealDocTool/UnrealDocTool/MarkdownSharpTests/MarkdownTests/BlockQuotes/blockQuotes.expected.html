<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" dir="ltr" lang="en-US">
	<head>
	</head>
	<body>
        <blockquote>
            <p>
                Block quote first.
            </p>
            <ol >
                <li>
                    <p>
                        List in quote.
                    </p>
                </li>

                <li>
                    <p>
                        Second list item.
                    </p>
                </li>
                
                <li>
                    <p>
                        Third list item.
                    </p>
                </li>
            </ol>
            <blockquote>
                <p>
                    Nested quote
                </p>
                <p>
                    Nested quote second paragraph.
                </p>
                <ul >
                    <li>
                        <p>
                            Nested quote list.
                        </p>
                    </li>

                    <li>
                        <p>
                            Second bullet.
                        </p>
                    </li>
                </ul>
            </blockquote>
            <p>
                Ending paragraph.
            </p>
        </blockquote>
	</body>
</html>
